docs/synthetics/browser-monitor.md

```diff
@@ -0,0 +1,104 @@
+title: Browser Monitor
+description: Browser monitor can monitor website performance and user journey.
+
+The browser monitor executes the configured script in a Chrome browser. It records various performance metrics during the execution. The script can extract & verify the page content using `assert` API during the execution. It can optionally collect screenshots.
+
+### Configuration
+
+#### General
+
+* **Name** - Name of the monitor. Max Length is 255 characters
+* **Frequency** - Frequency to execute the monitor
+* **Locations** - List of locations to run the monitor
+
+#### Script
+
+The browser monitor scripts are Node.js scripts that control a headless Chrome browser and use [Google Puppeteer](https://github.com/puppeteer/puppeteer#puppeteer) framework to drive the browser. For every run, the monitor will invoke the `testScript()` method with Puppeteer [Page](https://github.com/puppeteer/puppeteer/blob/master/docs/api.md#class-page) object as a parameter. The script content should be inside the `testScript()` method. For more information on specific use cases, you can refer to the `Browser Examples` section while creating a browser monitor. Use Node.js [assert](https://nodejs.org/api/assert.html) API to check if the values in the page are matching the requirements. If any assertion fails, the system declares the run as a failure. The logs will contain the details of the failure.
```

Comment:
you can refer => refer
are matching the => match your

The logs will contain the details of the failure => Monitor run logs, shown in Sematext, contain failure details